src/store/middleware/model-poller.test.ts

```diff
@@ -293,6 +294,7 @@ describe("model poller", () => {
   });
 
   it("updates the controller features if JIMM >= 4", async () => {
+    localStorage.setItem("rebac", JSON.stringify("true"));
```

Comment:
You don't have to use `JSON.stringify` here as "true" is already a string.

```suggestion
    localStorage.setItem("rebac", "true");
```